lib/Elastica/Aggregation/TopHits.php

```diff
@@ -66,7 +66,7 @@ public function setSort(array $sortArgs)
     /**
      * Allows to control how the _source field is returned with every hit.
      *
-     * @param array|bool $params Fields to be returned or false to disable source
+     * @param array|string|bool $params Fields to be returned or false to disable source
```

Comment:
Interesting. I assume you tested that this works? Could we add a test case?

Reply:
I'll tried to, and yeah, I built my app that way but phpstan told me I couldn't